oaiharvest/harvest.py

```diff
@@ -303,6 +304,8 @@ def main(argv=None):
         except Exception as e:
             # Log error
             logger.error(str(e))
```

Comment:
I think that you could achieve this with:

```python
    logger.error(str(e), exc_info=True)
```

This should mean that the traceback goes to the same place as the other log lines, e.g. if the logger were to get a FileHandler added to it, rather than to STDOUT/STDERR as it would with `traceback.print_exc()`